releases/release-6.1.6.md

```diff
@@ -0,0 +1,98 @@
+---
+title: TiDB 6.1.6 Release Notes
+summary: Learn about the compatibility changes, improvements, and bug fixes in TiDB 6.1.6.
+---
+
+# TiDB 6.1.6 Release Notes
+
+Release date: April 12, 2023
+
+TiDB version: 6.1.6
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.1/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v6.1/production-deployment-using-tiup) | [Installation packages](https://www.pingcap.com/download/?version=v6.1.6#version-list)
+
+## Compatibility changes
+
+- TiCDC fixes the issue of incorrect encoding of FLOAT data in Avro [#8490](https://github.com/pingcap/tiflow/issues/8490) @[3AceShowHand](https://github.com/3AceShowHand)
+
+    When upgrading the TiCDC cluster to v6.1.6 or later versions of v6.1.x, if a table replicated using Avro contains the `FLOAT` data type, you need to manually adjust the compatibility policy of Confluent Schema Registry to `None` before upgrading so that the changefeed can successfully update the schema. Otherwise, after upgrading, the changefeed will be unable to update the schema and enter an error state.
```

Comment:
```suggestion
    When upgrading the TiCDC cluster to v6.1.6 or a later v6.1.x version, if a table replicated using Avro contains the `FLOAT` data type, you need to manually adjust the compatibility policy of Confluent Schema Registry to `None` before upgrading so that the changefeed can successfully update the schema. Otherwise, after upgrading, the changefeed will be unable to update the schema and enter an error state.
```